### PR TITLE
beast sentence builder

### PR DESCRIPTION
**Pull request by SylvanSign**
http://54.174.101.223/ to test it out

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -15,20 +15,20 @@ const {
...
-const REQUIRED_KEYWORD_COUNT = 3
+const REQUIRED_KEYWORD_COUNT = 5
 
 const EXTRA_VALID_WORDS_REGEXP = /^[ai]$/
 
 let currentRoom = generateRoomName()
 let countDownTimer = null
 
-httpServer.listen(80)
+httpServer.listen(process.env.PORT || 80)
```
Comment (SylvanSign) [inline server.js]: this will let you start the app with `PORT=3000 gulp` on your mac, Zak

File: public/scripts/sentenceBuilder.js [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+      if (RANDOM_WORDS.includes(word)) {
+        // words.querySelector(`#word${i}`).className = 'keyword'
+        wordClasses[i] = 'keyword'
+        markRequiredWord(word)
+      } else {
+        indexesToCheck.push(i)
+      }
+    }
+  })
+  const wordsToCheck = wordsArray.filter((_, i) => indexesToCheck.includes(i))
```
Comment (SylvanSign) [inline public/scripts/sentenceBuilder.js:197]: `_` here is a convention where you're saying "I don't really care about that param". It's not special syntax or anything, `_` is just a valid identifier that look kind of cool as a throwaway name

File: server.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ let countDownTimer = null
 let votesNeeded
 let playerCount
 
-httpServer.listen(80)
+httpServer.listen(process.env.PORT || 80)
```
Comment (ZakMiller) [inline server.js:34]: What does this do?

File: server.js [hunk 4]
```diff
@@ -16,13 +16,13 @@ const {
 } = require('./rooms')
 const roundManager = require('./roundManager')
 
-const MIN_PLAYER_COUNT = 3
-const START_GAME_DELAY = 5 // sec
+const MIN_PLAYER_COUNT = 2
```
Comment (ZakMiller) [inline server.js]: woops

File: server.js (hunk 3 above)
Comment (SylvanSign) [reply server.js:34]: lets you run the game locally on whatever port you want by doing
```sh
PORT=3000 gulp

**Comment by SylvanSign** [comment]
fixed my woops

**Review by ZakMiller — APPROVED** [review]

**Pull request merged**